### PR TITLE
adding julia and python2 images

### PR DESCRIPTION
**Pull request by rportilla-databricks**
Tested these images with DBR 5.3

### CODE REVIEW

File: experimental/ubuntu/julia-0.6.4/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Start with DBR python virtual env image
+FROM databricksruntime/python-virtualenv:latest
```
Comment (evanye) [inline experimental/ubuntu/julia-0.6.4/Dockerfile]: is it required to start from the python base?

File: experimental/ubuntu/julia-0.6.4/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  && julia_home=$(ls -d /app/julia-* | grep -v "gz") \
+  && echo "$julia_home" \
+  && mv  $julia_home /app/julia \
+  && cd /app/julia/bin \
+  && export JULIA_HOME="$(pwd)" \
+  && echo "export JULIA_HOME=$(pwd)" | \tee -a /etc/profile \
+  && echo "export JULIA_HOME=$(pwd)" >> ~/.bashrc \
+  && echo "export PATH=$PATH:$JULIA_HOME" >> ~/.bashrc \
+  && ./julia -e "Pkg.update()" \
+  && export JULIA_HOME=$(pwd)
```
Comment (evanye) [inline experimental/ubuntu/julia-0.6.4/Dockerfile]: why do you need to export `JULIA_HOME` twice? Are these needed? The more docker way to set these is to specify `ENV JULIA_HOME=...`

File: experimental/ubuntu/julia-0.6.4/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  && tar xvfz julia-0.6.4-linux-x86_64.tar.gz \
+  && pwd \
+  && julia_home=$(ls -d /app/julia-* | grep -v "gz") \
+  && echo "$julia_home" \
+  && mv  $julia_home /app/julia \
+  && cd /app/julia/bin \
+  && export JULIA_HOME="$(pwd)" \
+  && echo "export JULIA_HOME=$(pwd)" | \tee -a /etc/profile \
+  && echo "export JULIA_HOME=$(pwd)" >> ~/.bashrc \
+  && echo "export PATH=$PATH:$JULIA_HOME" >> ~/.bashrc \
```
Comment (evanye) [inline experimental/ubuntu/julia-0.6.4/Dockerfile]: https://stackoverflow.com/questions/27093612/in-a-dockerfile-how-to-update-path-environment-variable

File: experimental/ubuntu/python2.7/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Start with DBR base image, all installations for Python REPL need to be installed
+FROM databricksruntime/minimal:latest
+
+# Installs python 2.7 and virtualenv for Spark and Notebooks
+RUN apt-get update \
+  && apt-get install -y \
+    python2.7 \
+    python-dev \
+    virtualenv \
+    python-subprocess32 \
```
Comment (evanye) [inline experimental/ubuntu/python2.7/Dockerfile]: is there a tldr as to why this package is needed?

File: experimental/ubuntu/julia-0.6.4/Dockerfile (hunk 1 above)
Comment (rportilla-databricks) [reply experimental/ubuntu/julia-0.6.4/Dockerfile]: What is in the python-virtualenv base image? If this is a standard set of libraries we use on databricks, then I think these will be preferred for a data scientist.

Comment (evanye) [reply experimental/ubuntu/julia-0.6.4/Dockerfile]: just python and some basic packages

File: experimental/ubuntu/python2.7/Dockerfile (hunk 4 above)
Comment (rportilla-databricks) [reply experimental/ubuntu/python2.7/Dockerfile]: I will add one, yes!

**Pull request merged**